_data/sidebars/docs_sidebar.yml

```diff
@@ -595,6 +595,10 @@ entries:
       url: /docs-meta-migration-guide.html
       output: web
 
+    - title: Content guidelines
+      url: /docs-meta-content-guidelines.html
+      output: web
+
```

Comment:
I think this can actually go directly after Overview. The migration guide is not particularly helpful to anyone anymore.

Reply:
Should we get rid of it?

Reply:
I think it does not hurt, but I am also fine with seeing it go.

Reply:
I'll move it to the bottom of the list.